### ISSUE TITLE
Export md file together with its attachment folder

### ISSUE THREAD

**Issue by cainiaoxiao** [nextcloud/text#3953]
**Is your feature request related to a problem? Please describe.**
This feature not related to problem.

**Describe the solution you'd like**
When a md document create with a picture, a folder create as well.Now user can only manual download folder manual.I think the user maybe can manual select if download picture floder at same time.

**Comment by mejo-** [comment]
Thanks a lot @cainiaoxiao for your report. I'm closing this as a duplicate of #102.

**Comment by max-nextcloud** [comment]
@mejo- You said you'd close this but then did not close...

I don't think this is a duplicate though. At least #102 does not talk about the possibility to export md file together with its attachment folder. This might be blocked by #102 as once an export function is in place this could be added as one of the export formats.
